src/streamvbyte.c

```diff
@@ -343,13 +73,82 @@ static uint8_t *svb_encode_scalar(const uint32_t *in,
 	return dataPtr; // pointer to first unused data byte
 }
 
+#ifdef __AVX__
+
+typedef union M128 {
+  char i8[16];
+  uint32_t u32[4];
+  __m128i i128;
+} u128;
+
+const u128 Ones = {.i8 = {1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1}};
```

Comment:
A global constant is unwarranted. This is literally used only in one function. It is much safer to declare it within the function where it is used.